src/hooks/useSafeReload.ts

```diff
@@ -0,0 +1,19 @@
+import { useEffect } from "react";
+
+const useSafeReload = () => {
```

Comment:
can you add some JSDoc here to briefly explain what this hook is for

Reply:
Done 👍